gtk/docs.md

```diff
@@ -1 +1,68 @@
-<!-- file * -->
+<!-- struct ImageMenuItem -->
+`[Deprecated since 3.10]` A `ImageMenuItem` is a menu item which has an icon next to the text label.
+
+This is functionally equivalent to:
+
+
+```C
+  GtkWidget *box = gtk_box_new (GTK_ORIENTATION_HORIZONTAL, 6);
+  GtkWidget *icon = gtk_image_new_from_icon_name ("folder-music-symbolic", GTK_ICON_SIZE_MENU);
+  GtkWidget *label = gtk_label_new ("Music");
+  GtkWidget *menu_item = gtk_menu_item_new ();
+
+  gtk_container_add (GTK_CONTAINER (box), icon);
+  gtk_container_add (GTK_CONTAINER (box), label);
+
+  gtk_container_add (GTK_CONTAINER (menu_item), box);
+
+  gtk_widget_show_all (menu_item);
+```
```

Comment:
I wanted to avoid modifying original docs as much as possible. However it seems useless now. I'll remove it.